spec/index.html

```diff
@@ -367,6 +367,42 @@ <h2>Introduction</h2>
       <p>There is also a [[[SPARQL12-RESULTS-XML]]] which follows a similar design pattern but uses XML as the
       serialization.</p>
       <p>Unless otherwise noted in the section heading, all sections and appendices in this document are normative.</p>
+
+	  <section id="version-announcement">
+	    <h3>Version Announcement</h3>
+	    <p>
+	    An optional version can be given to SPARQL results.
+	    </p>
+	    <p>
+	    The `version` <a href="#media-type">Media Type parameter</a> MAY be set to `1.2`
+	    to indicate possible use of RDF 1.2 features in the SPARQL results.
+        Alternatively, the <a href="#select-version">`"version"` member</a> MAY be used to specify this version.</p>
+        <p>When providing content over HTTP, servers can announce the version
+        using the optional `version` <a href="#media-type">Media Type parameter</a>:</p>
+
+        <pre id="ex-http-version-decl"
+             class="box http"
+             title="HTTP version announcement">GET /sparql/?query=PREFIX%20dc%3A%20%3Chttp%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2F%3E%20%0ASELECT%20%3Fbook%20%3Fwho%20%0AWHERE%20%7B%20%3Fbook%20dc%3Acreator%20%3Fwho%20%7D%0A HTTP/1.1
+Host: www.example
+Accept: application/sparql-results+json; version=1.2
+        </pre>
+
+        <p class="note">When using RDF 1.2-specific features, such as
+        <a data-cite="RDF12-CONCEPTS#dfn-base-direction">initial text direction</a>,
+        the specific RDF version can be announced using
+        either the `version` <a href="#media-type">Media Type parameter</a>
+        or the <a href="#select-version">`"version"` member</a> early in the document.
+        This allows parsers that do not support these features to detect
+        the presense of such features early, and potentially
+        inform the user, giving them an opportunity to stop the job
+        or otherwise act on the fact that some amount of the input data
+        will not be processed as desired.
+        </p>
+
+        <p class="note">Version numbers other than "1.2" are not defined by
+        this specification.
+        Processors MAY treat this as an error or warning.</p>
```

Comment:
MAY is normative, but a note is informative. Perhaps use “may” or “can”.

Reply:
I think this is the better choice.
```suggestion
        Processors can treat this as an error or warning.</p>
```